Source/WebCore/dom/Document.h

```diff
@@ -689,11 +691,11 @@ class Document
     bool needsStyleRecalc() const;
     unsigned lastStyleUpdateSizeForTesting() const { return m_lastStyleUpdateSizeForTesting; }
 
-    WEBCORE_EXPORT void updateLayout(OptionSet<LayoutOptions> = { }, const Element* = nullptr);
+    WEBCORE_EXPORT bool updateLayout(OptionSet<LayoutOptions> = { }, const Element* = nullptr);
```

Comment:
Without a comment, I am not sure it’s obvious from the name that the `bool` return value is a “did work” indicator.

Reply:
I agree that an enum return value would make the code more self-documenting.